### PR TITLE
Update registration text

### PR DESCRIPTION
**Pull request by kdeloach**

### CODE REVIEW

**Comment by kdeloach** [comment]
Is the URL to the census training just `/training`?

File: src/nyc_trees/apps/login/forms.py [hunk 1]
```diff
@@ -24,7 +25,7 @@ class ForgotUsernameForm(forms.Form):
 class NycRegistrationForm(RegistrationFormUniqueEmail):
     tos = forms.BooleanField(
         widget=forms.CheckboxInput,
-        label='I agree to the terms of use',
+        label=mark_safe('I agree to the <a href="http://www1.nyc.gov/home/terms-of-use.page" target="new">terms of use</a>'),  # NOQA
```
Comment (steventlamb) [inline src/nyc_trees/apps/login/forms.py]: No need to turn off linting. Python auto joins strings when they are separated only by whitespace.
```python
        label=mark_safe('I agree to the <a href="'
                        'http://www1.nyc.gov/home/terms-of-use.page'
                        '" target="new">terms of use</a>')
```

File: src/nyc_trees/templates/registration/activation_email.txt [hunk 2]
```diff
@@ -1,22 +1,15 @@
...
+Please click on this link to confirm your email address.
 
-{% blocktrans %}
-Sincerely,
-NYC TreesCount! Management
-{% endblocktrans %}
+http://{{site.domain}}{% url 'registration_activate' activation_key %}
 
+If you have additional questions or need help with your account, please email
+us at TreesCount.Help@parks.nyc.gov
```
Comment (steventlamb) [inline src/nyc_trees/templates/registration/activation_email.txt]: should we have a mailto anchor here?

File: src/nyc_trees/templates/registration/password_reset_form.html [hunk 3]
```diff
@@ -5,9 +5,7 @@
 
 {% block content %}
 <p>
-    {% blocktrans %}
-    Forgot your password?  Enter your email address or username in the form below and we'll send you instructions for creating a new one.
-    {% endblocktrans %}
+    Forgot your password? Enter your Trees Count! username or email address and click the “Reset” button below.
```
Comment (steventlamb) [inline src/nyc_trees/templates/registration/password_reset_form.html]: "Enter your Trees Count!" -> "Enter your username!" ?

Comment (steventlamb) [inline src/nyc_trees/templates/registration/password_reset_form.html]: oh, I hadn't read the whole line. Looks weird without any quoting around `Trees Count!`

**Comment by steventlamb** [comment]
+1

File: src/nyc_trees/templates/registration/activation_email.txt (hunk 2 above)
Comment (kdeloach) [inline src/nyc_trees/templates/registration/activation_email.txt]: I used an anchor in the HTML version (this is the text-only message).

**Comment by kdeloach** [comment]
Thanks, I made those updates.

**Pull request merged**